app/src/main/java/org/p2p/wallet/alarmlogger/logger/AlarmErrorsLogger.kt

```diff
@@ -53,7 +54,7 @@ class AlarmErrorsLogger(
         recipientAddress: SearchResult,
         error: Throwable
     ) {
-        if (!isLoggerEnabled) return
+        if (!isLoggerEnabled || error.shouldNotBeLogged) return
```

Comment:
I believe it's time to create wrapper which will do all boilerplate logic

At least this way
```kotlin
fun sendAlarm(block: suspend ()->Unit) {
     if (!isLoggerEnabled || error.shouldNotBeLogged) return
     appScope.launch{
         try { 
            block() 
          } catch(e: Throwable) {  
            Timber.e(AlarmErrorsError(error), "Failed to send alarm") 
          }
           
    }
}

fun triggerSendAlarm(
        token: Token.Active,
        currencyMode: CurrencyMode,
        amount: String,
        feePayerToken: Token.Active,
        accountCreationFee: String?,
        transactionFee: String?,
        relayAccount: RelayAccount,
        recipientAddress: SearchResult,
        error: Throwable
) = sendAlarm {
  // do whatever you want
  val request = getConverter<AlarmSendErrorConverter>()
                    .toSendErrorRequest(
                        token = token,
                        currencyMode = currencyMode,
                        amount = amount,
                        feePayerToken = feePayerToken,
                        accountCreationFee = accountCreationFee,
                        transactionFee = transactionFee,
                        relayAccount = relayAccount,
                        userPublicKey = userPublicKey,
                        recipientAddress = recipientAddress,
                        error = error
                    )
                retryRequest(
                    block = { api.sendAlarm(request) }
                )
}
```

But as I see there are just request creation, so sendAlarm could even get request from and sent it on its own